Cue/app/actions/library.js

```diff
@@ -17,4 +17,16 @@ async function loadLibrary(): PromiseAction {
   };
 }
 
-module.exports = { loadLibrary };
+async function setLibrary(library): PromiseAction {
```

Comment:
Uhm its really similar... I just realized I should just dispatch `loadLibrary`, bear with me :wink: